### PR TITLE
[15.0][MIG]website_event_questions_by_ticket: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
It seems there's no migration commit, but everything on the pre-commit one.

**Comment by bizzappdev** [comment]
> It seems there's no migration commit, but everything on the pre-commit one.

It is in testing from our side and that is why it is in WIP. but mostly no further changes are needed to make it work in v15.
@pedrobaeza

**Comment by bizzappdev** [comment]
ready for the review

**Review by stefan-tecnativa — APPROVED** [review]
Functionality tested 👍️

Just squash administrative commits and and all will be great.

File: website_event_questions_by_ticket/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Conditional Events Questions",
     "summary": "Events Questions conditional to the chosen ticket",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
```
Comment (chienandalu) [inline website_event_questions_by_ticket/__manifest__.py:7]: Your migration commit message should be:

[MIG] website_event_questions_by_ticket: Migration to 15.0

**Comment by stefan-tecnativa** [comment]
@bizzappdev what is the state of this task and this one #290 ??

**Comment by bizzappdev** [comment]
> @bizzappdev what is the state of this task and this one #290 ??

It is also ready for the review. 
and change for this MR is also done

**Comment by stefan-tecnativa** [comment]
@bizzappdev can you please change the commit message to the one suggested? 
`[MIG] website_event_questions_by_ticket: Migration to 15.0`

**Comment by pedrobaeza** [comment]
As there's no significant change to be made, but the commit message is incorrect, superseding it by #307

**Pull request closed without merging**